incubator/jaeger/values.yaml

```
@@ -40,6 +40,7 @@ storage:
     user: user
     usePassword: true
     password: password
+    tls: false
```

Maybe something like:

```suggestion
    tls:
      enabled: false
      ## Following pre-existing secret should contain `key`, `cert` and `ca` keys
      secretName: pre-existing-cassandra-tls-secret
```

And then in the templates wrap the env vars (..._TLS, ..._CERT, ..._KEY, ..._CA) around an if for `tls.enabled`, point them to the mounted secret. Also we should do the needful with regards to schema, see https://www.jaegertracing.io/docs/1.15/deployment/#tls-support for more details

---

@pavelnikolov care to chime in on this?

---

Yes, I'd prefer a preexisting secret.